### ISSUE TITLE
Sugar free granola bars

### ISSUE THREAD

**Issue by iteles** [dwyl/health#197]
I think these would be particularly tasty with hazelnut butter, which we have a lot of at the moment:

https://www.yummytoddlerfood.com/best-soft-granola-bars/

+ [ ] 1½ cups rolled oats
+ [ ] ½ cup shredded unsweetened coconut (or use ½ cup additional oats)
+ [ ] ¼ cup honey (or maple syrup or **apple sauce**)
+ [ ] ¼ cup whole-wheat flour
+ [ ] 3 tablespoons peanut butter (or sunflower seed butter or 1 egg)
+ [ ] 1 teaspoon vanilla extract
+ [ ] ½ teaspoon cinnamon
+ [ ] ⅛ teaspoon baking powder
+ [ ] ⅛ teaspoon salt
+ [ ] 2 tablespoons chia seeds
+ [ ] ¼ cup water (or ¼ cup mashed ripe banana, pumpkin puree, applesauce, butternut squash puree, OR sweet potato puree)

<img width="620" alt="image" src="https://github.com/user-attachments/assets/9038073c-3d0b-4fbe-8356-2e833ccc418d">

**Comment by nelsonic** [comment]
Indeed. Looks good. I've made something similar in the past. Do we have Apple Sauce in the house right now? 🍎 

_Full_ recipe with prep + baking instructions to avoid having to click through and scroll: 

![Screenshot 2024-09-01 at 07-09-06 Easy Homemade Granola Bars - Yummy Toddler Food](https://github.com/user-attachments/assets/5162a74d-335d-498f-b75e-dd618aa01eaf)

**Comment by iteles** [comment]
Made these today and they're just a bit uninteresting. I subbed applesauce throughout because our bananas were pretty green so I think next time maybe banana or a bit more salt. They need more sweetness and a more complex flavour.

A1 said they were just OK. A2 will eat any carb.
